### ISSUE TITLE
Crashes when manually setting pitches with no field selected

### ISSUE THREAD

**Issue by kclisp** [IllDepence/anki_add_pitch_plugin#27]
### Steps to reproduce
1. Browse your deck (just click on browse)
2. "set pitch accent" button is clickable
3. Click on "set pitch accent" and go through the dialogs
4. At the end, it crashes

### Expected result
Button should be greyed out and not clickable.

### Actual result
Button is clickable and produces a crash when used.

### Additional info
```
Debug info:
Anki 2.1.49 (dc80804a) Python 3.10.2 Qt 5.15.2 PyQt 5.15.6
Platform: Linux
Flags: frz=False ao=True sv=2
Add-ons, last update check: 2022-03-23 03:06:50
Add-ons possibly involved: ⁨Japanese Pitch Accent⁩

Caught exception:
Traceback (most recent call last):
  File "/home/kenny/.local/lib/python3.10/site-packages/aqt/webview.py", line 41, in cmd
    return json.dumps(self.onCmd(str))
  File "/home/kenny/.local/lib/python3.10/site-packages/aqt/webview.py", line 142, in _onCmd
    return self._onBridgeCmd(str)
  File "/home/kenny/.local/lib/python3.10/site-packages/aqt/webview.py", line 595, in _onBridgeCmd
    return self.onBridgeCmd(cmd)
  File "/home/kenny/.local/lib/python3.10/site-packages/aqt/editor.py", line 438, in onBridgeCmd
    self._links[cmd](self)
  File "/home/kenny/.local/share/Anki2/addons21/148002038/__init__.py", line 160, in set_pitch_dialog
    old_field_val = data[editor.web.editor.currentField][1]
TypeError: list indices must be integers or slices, not NoneType
```

**Comment by kclisp** [comment]
Or disallow manually setting pitches another way by saying that a field needs to be selected, if it's difficult to change the clickability of buttons

**Comment by IllDepence** [comment]
Thanks for the issue report and PR.

I’m still mainly using 2.1.44 and waiting for 2.1.50 because of the code editor problem (cf. #25).
In 2.1.44 the button is greyed out as long as no field is selected. But I could confirm the problem in 2.1.49.

Fixed in bf01e59.

**Comment by IllDepence** [comment]
Updated version including the fix to #26 and #27 is now [up on AnkiWeb](https://ankiweb.net/shared/info/148002038).

@kclisp I’ve put an attribution on the changelog as follows
> 2022/03/26 -- Fixed bugs concerning functionality to manually add/edit/remove annotations. (Thanks kclisp)

Let me know in case you’d want me to change or remove it.

**Comment by kclisp** [comment]
Looks good to me :)
